src/parser/parser_driver.cpp

```diff
@@ -375,6 +375,9 @@ void ParserDriver::defineTokens()
 
 		return std::make_pair(range, range);
 	});
+	_parser.token(R"(\\x[a-zA-Z0-9]{2})").states("$regexp").symbol("REGEXP_ESCAPE").description("regexp escaped character").action([](std::string_view str) -> Value {
```

Comment:
Matches hex escaped characters, like  "\x61", as a single token.

Reply:
Should we care about other encoding methods as well? It seems like `\65?` is a valid regex as well, not sure about the YARA side though 🤔

Reply:
I *think* yara doesn't support that:
```
rule kot {
    strings: $a = /kot\61kot/
    condition: $a
}
```
```
❯ yara -rs kot.yar /tmp/a.txt
kot.yar(2): error: invalid regular expression "$a": backreferences are not allowed
```
But sure, we should check for any other alternative encodings (if they exist). By the way I've checked, and `\X61` is not a valid encoding (`x` must be lowercase).